src/main/java/jpadb/service/DaoService.java

```diff
@@ -0,0 +1,21 @@
+package jpadb.service;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+public class DaoService {
```

Comment:
If it util class it should be final and have a private constructor. 
For lombok: https://projectlombok.org/features/experimental/UtilityClass